src/tests/validity/command_inception.ab

```diff
@@ -1,4 +1,4 @@
 // Output
 // Hello World
 
-$echo {$echo {$echo {$echo Hello World$ failed {}}$ failed {}}$ failed {}}$ failed {}
+$ echo { $ echo { $ echo { $ echo Hello World $ failed { } } $ failed { } } $ failed { } } $ failed { }
```

Comment:
Note, this outputs "Hello World" as expected, but shows "Empty failed block" warnings.